apps/ecommerce/frontend/src/components/Common/ProductCard/ProductCardBody/ProductCardBody.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Flex, Grid, Text } from '@contentful/f36-components';
+import { ExternalResourceError } from 'types';
+
+interface ProductCardProps {
+  title?: string;
```

Comment:
Similar comment here - it seems weird that all these props are optional when the ProductCard itself conceptually definitely should have props that must be required, like a title and description for example.

However, an image being optional makes sense to me.

Also some of these props if undefined will just print nothing at all - is this what we want? I suspect there is a more wholeistic design here if you make title and description required and then go from there.

Reply:
yeah this is a good point. Tbh these types all extend from the original `ExternalResource` type that we have throughout the application thus far. I somewhat assumed given how in flux everything is, that these were all designated as optional and as part of the iterative process will be refined as required. I am happy to make that change as part of this effort and require a title and description because I think that logically makes sense. My understanding since joining the project is that this is more about "when", not an "if" we would make this change. Happy to adjust 👍

Reply:
also within this component specifically, there might exist an error, and therefore the title and description might not be provided.

Reply:
Hmm okay, given the news we got yesterday in standup - to me it's just a matter of time/effort/value evaluation since this work will be handed off. I think you'll have the best judgement here on whether or not we should or should not clean this up in this PR.